_vector-search/ai-search/hybrid-search/collapse.md

```diff
@@ -0,0 +1,453 @@
+{ "index": {} }
+{ "item": "Vanilla Cake", "category": "cakes", "price": 16, "baked_date": "2023-07-03T00:00:00Z" }
+{ "index": {} }
+{ "item": "Vanilla Cake", "category": "cakes", "price": 17, "baked_date": "2023-07-09T00:00:00Z" }
+```
+
+Create a search pipeline:
+```json
+PUT /_search/pipeline/norm-pipeline
+{
+  "description": "Normalization processor for hybrid search",
+  "phase_results_processors": [
+    {
+      "normalization-processor": {
+        "normalization": {
+          "technique": "min_max"
+        },
+        "combination": {
+          "technique": "arithmetic_mean"
+        }
+      }
+    }
+  ]
+}
+```
+
+Search the index, collapsing on the `item` field:
+```json
+GET /bakery-items/_search?search_pipeline=norm-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "match": {
+            "item": "Chocolate Cake"
+          }
+        },
+        {
+          "bool": {
+            "must": {
+              "match": {
+                "category": "cakes"
+              }
+            }
+          }
+        }
+      ]
+    }
+  },
+  "collapse": {
+    "field": "item"
+  }
+}
+```
+
+Response:
+```json
+{
+  "took": 5,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 5,
+      "relation": "eq"
+    },
+    "max_score": 1.0,
+    "hits": [
+      {
+        "_index": "bakery-items",
+        "_id": "wBRPZZcB49c_2-1rYmO7",
+        "_score": 1.0,
+        "_source": {
+          "item": "Chocolate Cake",
+          "category": "cakes",
+          "price": 15,
+          "baked_date": "2023-07-01T00:00:00Z"
+        },
+        "fields": {
+          "item": [
+            "Chocolate Cake"
+          ]
+        }
+      },
+      {
+        "_index": "bakery-items",
+        "_id": "whRPZZcB49c_2-1rYmO7",
+        "_score": 0.5005,
+        "_source": {
+          "item": "Vanilla Cake",
+          "category": "cakes",
+          "price": 12,
+          "baked_date": "2023-07-02T00:00:00Z"
+        },
+        "fields": {
+          "item": [
+            "Vanilla Cake"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+Collapse is compatible with other features in hybrid query using syntax that is standard for those features, such as sort, pagination, and explain.
+For example, this is how you can combine collapse and sort:
+
+```json
+GET /bakery-items/_search?search_pipeline=norm-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "match": {
+                "item": "Chocolate Cake"
+          }
+        },
+        {
+          "bool": {
+                "must": {
+                    "match": {
+                        "category": "cakes"
+                    }
+                }
+          }
+        }
+      ]
+    }
+  },
+  "collapse": {
+    "field": "item"
+  },
+  "sort": "price"
```

Comment:
Nit: You can add order_by within the sort clause

Reply:
This is the same as the example in the main collapse documentation, I think we should keep it for the simplicity